samples/bluetooth/throughput/sample.yaml

```diff
@@ -7,4 +7,9 @@ tests:
     build_on_all: true
     platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840
       nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuappns
+    integration_platforms:
```

Comment:
Is "nrf51dk_nrf51422" intentionally omitted in integration_platforms?

Reply:
Fixed. `nrf51dk_nrf51422` should be removed from `platform_allow`